ol_schema/role/role.go

```diff
@@ -34,7 +36,7 @@ func Schema() map[string]*schema.Schema {
 // Inflate takes a key/value map of interfaces and uses the fields to construct a Role
 func Inflate(s map[string]interface{}) roles.Role {
 	out := roles.Role{}
-	if id, notNil := s["id"].(int); notNil {
+	if id, err := strconv.Atoi(s["id"].(string)); err == nil {
```

Comment:
The other implementations add `if s["id"] != nil {` check around this. Do you think it is unnecessary? Otherwise, please add that here.